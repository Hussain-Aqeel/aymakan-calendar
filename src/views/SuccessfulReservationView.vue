<template>
  <div class="container mx-auto text-gray-700 min-h-[60vh]">
    <div class="flex flex-col items-center">
      <img src="images/Sent-Email-color.svg"
           alt="email is sent"
           class=" h-60 w-60 lg:w-80 lg:h-80">
      <div class="p-8 mt-10 text-lg text-center">
        <p class="mb-4">Your next meeting is successfully reserved.</p>
        <p>You will receive an <strong>email</strong> with the reservation
          details.</p>
        <router-link to="/calendar" class="inline-block px-6 py-2 mt-10 bg-amber-400 rounded-xl hover:bg-amber-300">View Calendar</router-link>
      </div>

    </div>
  </div>
</template>

<script>
export default {

}
</script>

<style>

</style>