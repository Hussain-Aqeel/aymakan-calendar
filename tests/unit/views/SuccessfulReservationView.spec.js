import { shallowMount } from '@vue/test-utils';
import SuccessfulReservationView from '@/views/SuccessfulReservationView.vue';

describe('SuccessfulReservationView.vue', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(SuccessfulReservationView)
  })

  test('snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})