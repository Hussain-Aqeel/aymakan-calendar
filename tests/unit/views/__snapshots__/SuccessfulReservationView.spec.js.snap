// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessfulReservationView.vue snapshot 1`] = `
VueWrapper {
  "__app": Object {
    "_component": Object {
      "name": "VTU_ROOT",
      "render": [Function],
    },
    "_container": <div
      data-v-app=""
    >
      <div
        class="container mx-auto text-gray-700 min-h-[60vh]"
      >
        <div
          class="flex flex-col items-center"
        >
          <img
            alt="email is sent"
            class="h-60 w-60 lg:w-80 lg:h-80"
            src="images/Sent-Email-color.svg"
          />
          <div
            class="mt-10 p-8 text-lg text-center"
          >
            <p
              class="mb-4"
            >
              Your next meeting is successfully reserved.
            </p>
            <p>
              You will receive an 
              <strong>
                email
              </strong>
               with the reservation details.
            </p>
            <router-link
              class="inline-block mt-10 py-2 px-6 bg-amber-400 rounded-xl hover:bg-amber-300"
              to="/calendar"
            >
              View Calendar
            </router-link>
          </div>
        </div>
      </div>
    </div>,
    "_context": Object {
      "app": [Circular],
      "components": Object {
        "transition": Object {
          "name": "transition",
        },
        "transition-group": Object {
          "name": "transition-group",
        },
      },
      "config": Object {
        "compilerOptions": Object {},
        "errorHandler": undefined,
        "globalProperties": Object {},
        "isNativeTag": [Function],
        "optionMergeStrategies": Object {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": Object {},
      "emitsCache": WeakMap {},
      "mixins": Array [
        Object {
          "beforeCreate": [Function],
        },
      ],
      "optionsCache": WeakMap {},
      "propsCache": WeakMap {},
      "provides": Object {},
      "reload": [Function],
    },
    "_instance": Object {
      "a": null,
      "accessCache": Object {},
      "appContext": Object {
        "app": [Circular],
        "components": Object {
          "transition": Object {
            "name": "transition",
          },
          "transition-group": Object {
            "name": "transition-group",
          },
        },
        "config": Object {
          "compilerOptions": Object {},
          "errorHandler": undefined,
          "globalProperties": Object {},
          "isNativeTag": [Function],
          "optionMergeStrategies": Object {},
          "performance": false,
          "warnHandler": undefined,
        },
        "directives": Object {},
        "emitsCache": WeakMap {},
        "mixins": Array [
          Object {
            "beforeCreate": [Function],
          },
        ],
        "optionsCache": WeakMap {},
        "propsCache": WeakMap {},
        "provides": Object {},
        "reload": [Function],
      },
      "asyncDep": null,
      "asyncResolved": false,
      "attrs": Object {},
      "bc": null,
      "bm": null,
      "bu": null,
      "bum": null,
      "c": null,
      "components": null,
      "ctx": Object {},
      "da": null,
      "data": Object {},
      "directives": null,
      "ec": null,
      "effect": ReactiveEffect {
        "active": true,
        "allowRecurse": true,
        "deps": Array [
          Set {
            [Circular],
          },
          Set {
            [Circular],
          },
          Set {
            [Circular],
          },
        ],
        "fn": [Function],
        "onTrack": undefined,
        "onTrigger": undefined,
        "parent": undefined,
        "scheduler": [Function],
      },
      "emit": [Function],
      "emitsOptions": null,
      "emitted": null,
      "exposeProxy": null,
      "exposed": null,
      "inheritAttrs": undefined,
      "isDeactivated": false,
      "isMounted": true,
      "isUnmounted": false,
      "m": null,
      "next": null,
      "parent": null,
      "props": Object {},
      "propsDefaults": Object {},
      "propsOptions": Array [
        Object {},
        Array [],
      ],
      "provides": Object {},
      "proxy": Object {},
      "refs": Object {
        "VTU_COMPONENT": Object {
          "hasOwnProperty": [Function],
        },
      },
      "render": [Function],
      "renderCache": Array [],
      "root": [Circular],
      "rtc": null,
      "rtg": null,
      "scope": EffectScope {
        "active": true,
        "cleanups": Array [],
        "effects": Array [
          ReactiveEffect {
            "active": true,
            "allowRecurse": true,
            "deps": Array [
              Set {
                [Circular],
              },
              Set {
                [Circular],
              },
              Set {
                [Circular],
              },
            ],
            "fn": [Function],
            "onTrack": undefined,
            "onTrigger": undefined,
            "parent": undefined,
            "scheduler": [Function],
          },
        ],
      },
      "setupContext": null,
      "setupState": Object {},
      "slots": Object {},
      "sp": null,
      "subTree": Object {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": null,
        "children": null,
        "component": Object {
          "a": null,
          "accessCache": Object {
            "@@__IMMUTABLE_ITERABLE__@@": 0,
            "@@__IMMUTABLE_RECORD__@@": 0,
            "_isMockFunction": 0,
            "constructor": 0,
            "hasOwnProperty": 4,
            "isVueInstance": 0,
            "toJSON": 0,
            Symbol(Symbol.toStringTag): 0,
          },
          "appContext": Object {
            "app": [Circular],
            "components": Object {
              "transition": Object {
                "name": "transition",
              },
              "transition-group": Object {
                "name": "transition-group",
              },
            },
            "config": Object {
              "compilerOptions": Object {},
              "errorHandler": undefined,
              "globalProperties": Object {},
              "isNativeTag": [Function],
              "optionMergeStrategies": Object {},
              "performance": false,
              "warnHandler": undefined,
            },
            "directives": Object {},
            "emitsCache": WeakMap {},
            "mixins": Array [
              Object {
                "beforeCreate": [Function],
              },
            ],
            "optionsCache": WeakMap {},
            "propsCache": WeakMap {},
            "provides": Object {},
            "reload": [Function],
          },
          "asyncDep": null,
          "asyncResolved": false,
          "attrs": Object {},
          "bc": null,
          "bm": null,
          "bu": null,
          "bum": null,
          "c": null,
          "components": Object {},
          "ctx": Object {
            "hasOwnProperty": [Function],
          },
          "da": null,
          "data": Object {},
          "directives": null,
          "ec": null,
          "effect": ReactiveEffect {
            "active": true,
            "allowRecurse": true,
            "deps": Array [],
            "fn": [Function],
            "onTrack": undefined,
            "onTrigger": undefined,
            "parent": undefined,
            "scheduler": [Function],
          },
          "emit": [Function],
          "emitsOptions": null,
          "emitted": null,
          "exposeProxy": null,
          "exposed": null,
          "inheritAttrs": undefined,
          "isDeactivated": false,
          "isMounted": true,
          "isUnmounted": false,
          "m": null,
          "next": null,
          "parent": [Circular],
          "props": Object {},
          "propsDefaults": Object {},
          "propsOptions": Array [
            Object {},
            Array [],
          ],
          "provides": Object {},
          "proxy": Object {
            "hasOwnProperty": [Function],
          },
          "refs": Object {},
          "render": [Function],
          "renderCache": Array [],
          "root": [Circular],
          "rtc": null,
          "rtg": null,
          "scope": EffectScope {
            "active": true,
            "cleanups": Array [],
            "effects": Array [
              ReactiveEffect {
                "active": true,
                "allowRecurse": true,
                "deps": Array [],
                "fn": [Function],
                "onTrack": undefined,
                "onTrigger": undefined,
                "parent": undefined,
                "scheduler": [Function],
              },
            ],
          },
          "setupContext": null,
          "setupState": Object {},
          "slots": Object {},
          "sp": null,
          "subTree": Object {
            "__v_isVNode": true,
            "__v_skip": true,
            "anchor": null,
            "appContext": null,
            "children": Array [
              Object {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": Array [
                  Object {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": null,
                    "component": null,
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <img
                      alt="email is sent"
                      class="h-60 w-60 lg:w-80 lg:h-80"
                      src="images/Sent-Email-color.svg"
                    />,
                    "key": null,
                    "patchFlag": -1,
                    "props": Object {
                      "alt": "email is sent",
                      "class": "h-60 w-60 lg:w-80 lg:h-80",
                      "src": "images/Sent-Email-color.svg",
                    },
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 1,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "img",
                  },
                  Object {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": Array [
                      Object {
                        "__v_isVNode": true,
                        "__v_skip": true,
                        "anchor": null,
                        "appContext": null,
                        "children": "Your next meeting is successfully reserved.",
                        "component": null,
                        "dirs": null,
                        "dynamicChildren": null,
                        "dynamicProps": null,
                        "el": <p
                          class="mb-4"
                        >
                          Your next meeting is successfully reserved.
                        </p>,
                        "key": null,
                        "patchFlag": -1,
                        "props": Object {
                          "class": "mb-4",
                        },
                        "ref": null,
                        "scopeId": null,
                        "shapeFlag": 9,
                        "slotScopeIds": null,
                        "ssContent": null,
                        "ssFallback": null,
                        "staticCount": 0,
                        "suspense": null,
                        "target": null,
                        "targetAnchor": null,
                        "transition": null,
                        "type": "p",
                      },
                      Object {
                        "__v_isVNode": true,
                        "__v_skip": true,
                        "anchor": null,
                        "appContext": null,
                        "children": Array [
                          Object {
                            "__v_isVNode": true,
                            "__v_skip": true,
                            "anchor": null,
                            "appContext": null,
                            "children": "You will receive an ",
                            "component": null,
                            "dirs": null,
                            "dynamicChildren": null,
                            "dynamicProps": null,
                            "el": You will receive an ,
                            "key": null,
                            "patchFlag": 0,
                            "props": null,
                            "ref": null,
                            "scopeId": null,
                            "shapeFlag": 8,
                            "slotScopeIds": null,
                            "ssContent": null,
                            "ssFallback": null,
                            "staticCount": 0,
                            "suspense": null,
                            "target": null,
                            "targetAnchor": null,
                            "transition": null,
                            "type": Symbol(Text),
                          },
                          Object {
                            "__v_isVNode": true,
                            "__v_skip": true,
                            "anchor": null,
                            "appContext": null,
                            "children": "email",
                            "component": null,
                            "dirs": null,
                            "dynamicChildren": null,
                            "dynamicProps": null,
                            "el": <strong>
                              email
                            </strong>,
                            "key": null,
                            "patchFlag": 0,
                            "props": null,
                            "ref": null,
                            "scopeId": null,
                            "shapeFlag": 9,
                            "slotScopeIds": null,
                            "ssContent": null,
                            "ssFallback": null,
                            "staticCount": 0,
                            "suspense": null,
                            "target": null,
                            "targetAnchor": null,
                            "transition": null,
                            "type": "strong",
                          },
                          Object {
                            "__v_isVNode": true,
                            "__v_skip": true,
                            "anchor": null,
                            "appContext": null,
                            "children": " with the reservation details.",
                            "component": null,
                            "dirs": null,
                            "dynamicChildren": null,
                            "dynamicProps": null,
                            "el":  with the reservation details.,
                            "key": null,
                            "patchFlag": 0,
                            "props": null,
                            "ref": null,
                            "scopeId": null,
                            "shapeFlag": 8,
                            "slotScopeIds": null,
                            "ssContent": null,
                            "ssFallback": null,
                            "staticCount": 0,
                            "suspense": null,
                            "target": null,
                            "targetAnchor": null,
                            "transition": null,
                            "type": Symbol(Text),
                          },
                        ],
                        "component": null,
                        "dirs": null,
                        "dynamicChildren": null,
                        "dynamicProps": null,
                        "el": <p>
                          You will receive an 
                          <strong>
                            email
                          </strong>
                           with the reservation details.
                        </p>,
                        "key": null,
                        "patchFlag": -1,
                        "props": null,
                        "ref": null,
                        "scopeId": null,
                        "shapeFlag": 17,
                        "slotScopeIds": null,
                        "ssContent": null,
                        "ssFallback": null,
                        "staticCount": 0,
                        "suspense": null,
                        "target": null,
                        "targetAnchor": null,
                        "transition": null,
                        "type": "p",
                      },
                      Object {
                        "__v_isVNode": true,
                        "__v_skip": true,
                        "anchor": null,
                        "appContext": null,
                        "children": Array [
                          Object {
                            "__v_isVNode": true,
                            "__v_skip": true,
                            "anchor": null,
                            "appContext": null,
                            "children": "View Calendar",
                            "component": null,
                            "dirs": null,
                            "dynamicChildren": null,
                            "dynamicProps": null,
                            "el": View Calendar,
                            "key": null,
                            "patchFlag": 0,
                            "props": null,
                            "ref": null,
                            "scopeId": null,
                            "shapeFlag": 8,
                            "slotScopeIds": null,
                            "ssContent": null,
                            "ssFallback": null,
                            "staticCount": 0,
                            "suspense": null,
                            "target": null,
                            "targetAnchor": null,
                            "transition": null,
                            "type": Symbol(Text),
                          },
                        ],
                        "component": null,
                        "dirs": null,
                        "dynamicChildren": null,
                        "dynamicProps": null,
                        "el": <router-link
                          class="inline-block mt-10 py-2 px-6 bg-amber-400 rounded-xl hover:bg-amber-300"
                          to="/calendar"
                        >
                          View Calendar
                        </router-link>,
                        "key": null,
                        "patchFlag": 0,
                        "props": Object {
                          "class": "inline-block mt-10 py-2 px-6 bg-amber-400 rounded-xl hover:bg-amber-300",
                          "to": "/calendar",
                        },
                        "ref": null,
                        "scopeId": null,
                        "shapeFlag": 17,
                        "slotScopeIds": null,
                        "ssContent": null,
                        "ssFallback": null,
                        "staticCount": 0,
                        "suspense": null,
                        "target": null,
                        "targetAnchor": null,
                        "transition": null,
                        "type": "router-link",
                      },
                    ],
                    "component": null,
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <div
                      class="mt-10 p-8 text-lg text-center"
                    >
                      <p
                        class="mb-4"
                      >
                        Your next meeting is successfully reserved.
                      </p>
                      <p>
                        You will receive an 
                        <strong>
                          email
                        </strong>
                         with the reservation details.
                      </p>
                      <router-link
                        class="inline-block mt-10 py-2 px-6 bg-amber-400 rounded-xl hover:bg-amber-300"
                        to="/calendar"
                      >
                        View Calendar
                      </router-link>
                    </div>,
                    "key": null,
                    "patchFlag": 0,
                    "props": Object {
                      "class": "mt-10 p-8 text-lg text-center",
                    },
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 17,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "div",
                  },
                ],
                "component": null,
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <div
                  class="flex flex-col items-center"
                >
                  <img
                    alt="email is sent"
                    class="h-60 w-60 lg:w-80 lg:h-80"
                    src="images/Sent-Email-color.svg"
                  />
                  <div
                    class="mt-10 p-8 text-lg text-center"
                  >
                    <p
                      class="mb-4"
                    >
                      Your next meeting is successfully reserved.
                    </p>
                    <p>
                      You will receive an 
                      <strong>
                        email
                      </strong>
                       with the reservation details.
                    </p>
                    <router-link
                      class="inline-block mt-10 py-2 px-6 bg-amber-400 rounded-xl hover:bg-amber-300"
                      to="/calendar"
                    >
                      View Calendar
                    </router-link>
                  </div>
                </div>,
                "key": null,
                "patchFlag": 0,
                "props": Object {
                  "class": "flex flex-col items-center",
                },
                "ref": null,
                "scopeId": null,
                "shapeFlag": 17,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "div",
              },
            ],
            "component": null,
            "dirs": null,
            "dynamicChildren": Array [],
            "dynamicProps": null,
            "el": <div
              class="container mx-auto text-gray-700 min-h-[60vh]"
            >
              <div
                class="flex flex-col items-center"
              >
                <img
                  alt="email is sent"
                  class="h-60 w-60 lg:w-80 lg:h-80"
                  src="images/Sent-Email-color.svg"
                />
                <div
                  class="mt-10 p-8 text-lg text-center"
                >
                  <p
                    class="mb-4"
                  >
                    Your next meeting is successfully reserved.
                  </p>
                  <p>
                    You will receive an 
                    <strong>
                      email
                    </strong>
                     with the reservation details.
                  </p>
                  <router-link
                    class="inline-block mt-10 py-2 px-6 bg-amber-400 rounded-xl hover:bg-amber-300"
                    to="/calendar"
                  >
                    View Calendar
                  </router-link>
                </div>
              </div>
            </div>,
            "key": null,
            "patchFlag": 0,
            "props": Object {
              "class": "container mx-auto text-gray-700 min-h-[60vh]",
            },
            "ref": null,
            "scopeId": null,
            "shapeFlag": 17,
            "slotScopeIds": null,
            "ssContent": null,
            "ssFallback": null,
            "staticCount": 0,
            "suspense": null,
            "target": null,
            "targetAnchor": null,
            "transition": null,
            "type": "div",
          },
          "suspense": null,
          "suspenseId": 0,
          "type": Object {
            "components": Object {},
            "render": [Function],
          },
          "u": null,
          "uid": 1,
          "um": null,
          "update": [Function],
          "vnode": [Circular],
          "withProxy": null,
        },
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": <div
          class="container mx-auto text-gray-700 min-h-[60vh]"
        >
          <div
            class="flex flex-col items-center"
          >
            <img
              alt="email is sent"
              class="h-60 w-60 lg:w-80 lg:h-80"
              src="images/Sent-Email-color.svg"
            />
            <div
              class="mt-10 p-8 text-lg text-center"
            >
              <p
                class="mb-4"
              >
                Your next meeting is successfully reserved.
              </p>
              <p>
                You will receive an 
                <strong>
                  email
                </strong>
                 with the reservation details.
              </p>
              <router-link
                class="inline-block mt-10 py-2 px-6 bg-amber-400 rounded-xl hover:bg-amber-300"
                to="/calendar"
              >
                View Calendar
              </router-link>
            </div>
          </div>
        </div>,
        "key": null,
        "patchFlag": 0,
        "props": Object {
          "ref": "VTU_COMPONENT",
        },
        "ref": Object {
          "f": false,
          "i": [Circular],
          "k": undefined,
          "r": "VTU_COMPONENT",
        },
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "transition": null,
        "type": Object {
          "components": Object {},
          "render": [Function],
        },
      },
      "suspense": null,
      "suspenseId": 0,
      "type": Object {
        "name": "VTU_ROOT",
        "render": [Function],
      },
      "u": null,
      "uid": 0,
      "um": null,
      "update": [Function],
      "vnode": Object {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": Object {
          "app": [Circular],
          "components": Object {
            "transition": Object {
              "name": "transition",
            },
            "transition-group": Object {
              "name": "transition-group",
            },
          },
          "config": Object {
            "compilerOptions": Object {},
            "errorHandler": undefined,
            "globalProperties": Object {},
            "isNativeTag": [Function],
            "optionMergeStrategies": Object {},
            "performance": false,
            "warnHandler": undefined,
          },
          "directives": Object {},
          "emitsCache": WeakMap {},
          "mixins": Array [
            Object {
              "beforeCreate": [Function],
            },
          ],
          "optionsCache": WeakMap {},
          "propsCache": WeakMap {},
          "provides": Object {},
          "reload": [Function],
        },
        "children": null,
        "component": [Circular],
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": <div
          class="container mx-auto text-gray-700 min-h-[60vh]"
        >
          <div
            class="flex flex-col items-center"
          >
            <img
              alt="email is sent"
              class="h-60 w-60 lg:w-80 lg:h-80"
              src="images/Sent-Email-color.svg"
            />
            <div
              class="mt-10 p-8 text-lg text-center"
            >
              <p
                class="mb-4"
              >
                Your next meeting is successfully reserved.
              </p>
              <p>
                You will receive an 
                <strong>
                  email
                </strong>
                 with the reservation details.
              </p>
              <router-link
                class="inline-block mt-10 py-2 px-6 bg-amber-400 rounded-xl hover:bg-amber-300"
                to="/calendar"
              >
                View Calendar
              </router-link>
            </div>
          </div>
        </div>,
        "key": null,
        "patchFlag": 0,
        "props": null,
        "ref": null,
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "transition": null,
        "type": Object {
          "name": "VTU_ROOT",
          "render": [Function],
        },
      },
      "withProxy": null,
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": Object {
      "compilerOptions": Object {},
      "errorHandler": undefined,
      "globalProperties": Object {},
      "isNativeTag": [Function],
      "optionMergeStrategies": Object {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "provide": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.2.37",
  },
  "__setProps": [Function],
  "componentVM": Object {
    "hasOwnProperty": [Function],
  },
  "isDisabled": [Function],
  "rootVM": Object {},
  "wrapperElement": <div
    class="container mx-auto text-gray-700 min-h-[60vh]"
  >
    <div
      class="flex flex-col items-center"
    >
      <img
        alt="email is sent"
        class="h-60 w-60 lg:w-80 lg:h-80"
        src="images/Sent-Email-color.svg"
      />
      <div
        class="mt-10 p-8 text-lg text-center"
      >
        <p
          class="mb-4"
        >
          Your next meeting is successfully reserved.
        </p>
        <p>
          You will receive an 
          <strong>
            email
          </strong>
           with the reservation details.
        </p>
        <router-link
          class="inline-block mt-10 py-2 px-6 bg-amber-400 rounded-xl hover:bg-amber-300"
          to="/calendar"
        >
          View Calendar
        </router-link>
      </div>
    </div>
  </div>,
}
`;
